cmd/contour/contour.go

```diff
@@ -61,6 +62,7 @@ func main() {
 	sds.Arg("resources", "SDS resource filter").StringsVar(&resources)
 
 	serve, serveCtx := registerServe(app)
+	version := app.Command("version", "Build information for Contour. Contains the branch and git SHA that Contour was built from, and the Contour version.")
```

Comment:
Shall we keep this short, just "Build information for Contour."

The difference being that you can just run it and see the output, which is not the case with the metric (you have to get it to serve, etc).

Reply:
Makes sense and keeps to a similar style for flags.

Reply:
Have updated to a shorter description.